test/test_transformers.py

```diff
@@ -724,6 +731,58 @@ def rand_tensor(*shape):
             if dropout_p == 0.0 or device == 'cpu':
                 self.assertEqual(actual, expected)
 
+    @unittest.skipIf(TEST_WITH_CROSSREF, 'Fastpath not available with crossref')
+    def test_mask_check_fastpath(self):
+        """
+        Test that fastpath is executed independently of the mask that is passed.
+        If the passed mask is left aligned, test that nested tensors are used (sparsity fastpath),
+        otherwise use fastpath with traditional tensors.
+        """
+
+        with torch.no_grad():
```

Comment:
style nit: you could alternatively decorate the test with `@torch.no_grad()`

Reply:
Nice!